src/main/java/org/json/JSONArray.java

```diff
@@ -2023,4 +2024,13 @@ private static JSONException wrongValueFormatException(
                 , cause);
     }
 
+    /**
+     * Returns a sequential {@code Stream} with this JSONArray as its source.
+     *
+     * @return a sequential {@code Stream} over the elements in this JSONArray
+     */
+    public Stream<Object> stream() {
+        return myArrayList.stream();
+    }
```

Comment:
There is already a `toList()` method available. We can call `toList().stream()`.